Eventnet/Eventnet.Api/Controllers/TokenController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Eventnet.DataAccess;
+using Eventnet.Models;
+using Eventnet.Services;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Eventnet.Controllers;
+
+[Route("api/token")]
+public class TokenController : Controller
+{
+    private readonly IJwtAuthService jwtAuthService;
+    private readonly UserManager<ApplicationUser> userManager;
+
+    public TokenController(UserManager<ApplicationUser> userManager,
+        IJwtAuthService jwtAuthService)
+    {
+        this.userManager = userManager;
+        this.jwtAuthService = jwtAuthService;
+    }
+
+    [Authorize]
+    [HttpGet("me")]
+    public async Task<IActionResult> GetCurrentUser()
+    {
+        var userName = User.Claims
```

Comment:
Дело вкуса, но я бы переписал так

```suggestion
        var userName = User.Claims
        .Where(x => x.Type == ClaimTypes.Name)
        .Select(x => x.Value)
        .FirstOrDefault();
```

либо

```suggestion
        var user = User.Claims.FirstOrDefault(x => x.Type == ClaimTypes.Name);
        var userName = user?.Value;
```